app/src/migrations/20200506163000-6-map-location-table.js

```diff
@@ -0,0 +1,68 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.createTable('map_location', {
+          mapLocationId: {
+            allowNull: false,
+            defaultValue: Sequelize.UUIDV4,
+            primaryKey: true,
+            type: Sequelize.UUID
+          },
+          locationId: {
+            allowNull: false,
+            comment: 'Associated location id',
+            type: Sequelize.UUID,
+            references: {
+              model: 'location',
+              key: 'locationId'
+            }
+          },
+          latitude: {
+            allowNull: false,
```

Comment:
I would consider not using varchars for this field. Best case for geography would be PostGIS but might not be important in the timeframe.
Could see what @parc-jason and @jujaga think too but there the point data type which might be perfect, or use appropriate precision decimal-type fields (since it's not nullable anyways).
Relevant: https://stackoverflow.com/a/8150944/324672

Reply:
For now, let's just use [numeric data types](https://sequelize.org/master/class/lib/data-types.js~NUMBER.html).  Check over Postgres definitions [too](https://www.postgresql.org/docs/9.1/datatype-numeric.html).

Reply:
good call Lucas. PostGIS looks great, I'm not sure we have the geography module on our postgres image. point also sounds good. i'll look into these later